### PR TITLE
Respect MinimumRange values when Minimum and Maximum values are also set

### LINKED ISSUE #550: MinimumRange gives odd result
I have a LineSeries where I have set the axis to have a minimum of 0. I have also set the minimumrange to be 10. With my data, that gives a display from -5 to 5. I think it would make more sense to use the minimum when deciding what the lower end of the range should be.

I tested this with the 2014.1.546 and 2015.1.982.

### PR DESCRIPTION
**Pull request by classicboss302**
Resolves #550.

### CODE REVIEW

File: Source/OxyPlot/Axes/Axis.cs [hunk 1]
```diff
@@ -722,10 +722,21 @@ public virtual void CoerceActualMaxMin()
             var range = this.ActualMaximum - this.ActualMinimum;
             if (range < this.MinimumRange)
             {
-                var average = (this.ActualMaximum + this.ActualMinimum) * 0.5;
-                var delta = this.MinimumRange / 2;
-                this.ActualMinimum = average - delta;
-                this.ActualMaximum = average + delta;
+                if (this.Minimum != double.NaN)
```
Comment (objorke) [inline Source/OxyPlot/Axes/Axis.cs]: isn't it better to use `!double.IsNan(this.Minimum)` for such NaN comparisons?

**Comment by objorke** [comment]
Thanks for contributing! I have a few comments:
1. Is this issue covered by tests/examples? If not, please add some models in the example library that demonstrates the problem. I will run these before merging the code.
2. Please update the change log an contributors files.
Please have a look at the [contributing guidelines](https://github.com/oxyplot/oxyplot/blob/develop/CONTRIBUTING.md) :-)

**Comment by classicboss302** [comment]
I think I have everything in place now. Thanks for the feedback.

**Comment by objorke** [comment]
I am starting a rebuild of this PR

File: Source/Examples/WPF/WpfExamples/Examples/MinRangeDemo/MainWindow.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    [Example("Shows OxyPlot with a Minimum Range and Minimum values.")]
+    public partial class MainWindow : Window
+    {
+        Random r = new Random(13);
+
+        public MainWindow()
+        {
+            this.InitializeComponent();
+
+            Model = GenerateRandomPlotModel(string.Format("MinRange plot"));
```
Comment (objorke) [inline Source/Examples/WPF/WpfExamples/Examples/MinRangeDemo/MainWindow.xaml.cs]: I would prefer this example to be added to the ExampleLibrary instead!

**Comment by objorke** [comment]
Please remove the MinRangeDemo and move the example PlotModel to the ExampleLibrary

**Comment by objorke** [comment]
The build error seems to be in the GitVersion step:
``` 
ERROR [10/17/15 6:53:09:73] An unexpected error occurred:
System.Net.WebException: The remote server returned an error: (500) Internal Server Error.
   at System.Net.HttpWebRequest.GetResponse()
   at GitVersion.AppVeyor.GenerateSetVersionMessage(String versionToUseForBuildNumber)
   at GitVersion.BuildServerBase.WriteIntegration(Action`1 writer, VersionVariables variables)
   at GitVersion.SpecifiedArgumentRunner.Run(Arguments arguments, IFileSystem fileSystem)
   at GitVersion.Program.VerifyArgumentsAndRun()
INFO [10/17/15 6:53:09:73] 
INFO [10/17/15 6:53:09:73] Here is the current git graph (please include in issue): 
INFO [10/17/15 6:53:09:73] Showing max of 100 com
```

File: Source/OxyPlot/Axes/Axis.cs [hunk 3]
```diff
@@ -722,10 +722,21 @@ public virtual void CoerceActualMaxMin()
             var range = this.ActualMaximum - this.ActualMinimum;
             if (range < this.MinimumRange)
             {
-                var average = (this.ActualMaximum + this.ActualMinimum) * 0.5;
-                var delta = this.MinimumRange / 2;
-                this.ActualMinimum = average - delta;
-                this.ActualMaximum = average + delta;
+                if (!double.IsNaN(this.Minimum))
+                {
+                    this.ActualMaximum = this.Minimum + this.MinimumRange;
```
Comment (objorke) [inline Source/OxyPlot/Axes/Axis.cs]: this is a breaking change, isn´t it?

Comment (objorke) [inline Source/OxyPlot/Axes/Axis.cs]: to keep the current behavior you could use the current code `if (!double.IsNaN(this.Minimum) && !double.IsNaN(this.Maximum))`, and use the new code in the two other cases

**Comment by classicboss302** [comment]
I have updated the pull request.

**Pull request merged**